### PR TITLE
Portfolio: Add new sections and more responsiveness

### PR DESCRIPTION
**Pull request by romans-adi**
For this pull request:

- I have added sections Works and About and styles to both sections according to the template design.
- Header elements were changed due to new elements have been added.
- I used grids and flexbox for most of the elements in new sections.
- Project is linted with WebHint, Lighthouse, and StyleLint.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,63 +5,235 @@
...
+          </div>
+          <h2 class="blue section-heading about-heading">About me</h2>
+          <div class="about-myself-text">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded,
+              don’t hesitate to contact me.</p>
+          </div>
+          <div class="about-myself-illustration-left">
+            <img src="assets/img/about-me-vector-left.png" alt="about-myself-illustration">
+          </div>
```
Comment (levy002) [inline index.html:179]: - [ ] Kindly make sure to insert this background image on the page using CSS background properties in order to follow HTML & CSS best practices and meet the project requirements. Kindly read [this lesson](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) on how to insert background-image using CSS.

File: index.html [hunk 2]
```diff
@@ -5,63 +5,235 @@
...
+            <a href="#" type="button" class="btn-success btn-see-project">See Project</a>
+          </div>
+        </div>
+      </section>
+
+      <section id="about-myself" class="about-myself">
+        <div class="about-myself-content">
+          <div class="about-myself-illustration-right">
+            <img src="assets/img/about-me-vector-right.png" alt="about-myself-illustration">
+          </div>
```
Comment (levy002) [inline index.html:170]: - [ ] Kindly make sure to insert this background image on the page using CSS background properties in order to follow HTML & CSS best practices and meet the project requirements. Kindly read [this lesson](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) on how to insert background-image using CSS.

File: assets/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+  padding: 6% 0;
+}
+
+.projects-set {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 5%;
+}
```
Comment (levy002) [inline assets/css/styles.css:276]: - [ ]  KIndly make sure to align the projects by using CSS Grid Layout in order to meet the project requirements. Please read these lessons ([lesson-1](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) and [lesson-2](https://css-tricks.com/snippets/css/complete-guide-grid/)) to understand how to align content using CSS GRID LAYOUT.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @romans-adi   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @romans-adi  👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Displayed the project section with CSS grid property ✔️
Made all required change ✔️
Used images and background-image to enhance the look of the website ✔️

## Optional suggestions

- [ ] Kindly fill in the `Show your support` section of the Readme file and also remove the template text from the `License` section. 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**